front/src/app/modules/input-login/input-login.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { authService } from 'src/app/core/services/auth.service';
+import { HttpClient } from '@angular/common/http';
+
+type email={
+  email:string
+}
+
+@Component({
+  selector: 'app-input-login',
+  templateUrl: './input-login.component.html',
+  styleUrls: ['./input-login.component.css']
+})
+export class InputLoginComponent implements OnInit {
+  email = new FormControl("",[Validators.email,Validators.required])
+  
+
+
+
+
+  constructor(private http: HttpClient,  private authservice: authService,  private router: Router) { }
+  ngOnInit(): void {
+    this.http.post('http://localhost:3000',{email:this.email}).subscribe();
```

Comment:
Esto no va aca, hace que cada vez que se abre la ventana de login se envie un peticion post automaticamente, y no queremos eso. La peticion solo se hace cuando el usuario da click en "Iniciar" y el formulario es valido. Ademas, this.email es un control de formulario, no el valor del campo email. Para acceder al valor debes usar this.email.value, pero eso ya lo sabes ;)

Porfa borra esta linea de codigo

Reply:
borro laline completa o  {email:this.email})
yo lo tenia asi  this.http.get('http://localhost:3000').subscribe();